stable/kubernetes-dashboard/templates/ingress.yaml

```diff
@@ -1,38 +1,38 @@
 {{- if .Values.ingress.enabled -}}
-{{- $fullName := include "kubernetes-dashboard.fullname" . }}
-{{- $httpPort := .Values.httpPort }}
+{{- $serviceName := include "kubernetes-dashboard.fullname" . -}}
```

Comment:
Here it is
```include "kubernetes-dashboard.fullname" . -```
while it is
```template "kubernetes-dashboard.service.fullname" .```
in the service.

This doesn't work for me (ing has "kubernetes-dashboard-pp-kubernetes-dashboard", svc ,name is "kubernetes-dashboard")

Reply:
Should be fixed in #3475.